src/Microsoft.AspNet.DataProtection/DataProtectionConfiguration.cs

```diff
@@ -114,6 +114,36 @@ public DataProtectionConfiguration ConfigureGlobalOptions([NotNull] Action<DataP
         }
 
         /// <summary>
+        /// Configures the data protection system not to generate new keys automatically.
+        /// </summary>
+        /// <returns>The 'this' instance.</returns>
+        /// <remarks>
+        /// Calling this API corresponds to setting <see cref="KeyManagementOptions.AutoGenerateKeys"/>
+        /// to 'false'. See that property's documentation for more information.
+        /// </remarks>
+        public DataProtectionConfiguration DisableAutomaticKeyGeneration()
```

Comment:
We don't really have this pattern anywhere else. A single method to configure individual properties of a single options model.